### ISSUE TITLE
"connect via bluetooth" can not be enabled

### ISSUE THREAD

**Issue by Goooyi** [rohitsangwan01/uni_control_hub#30]
first, thanks for your effort, on my win11 pc, I have started the default server successfully ,
here is the log output: 
```UniControlHub (v0.0.8) Logs: 

[2024-10-21T06:50:01] Info: Trying to Start
[2024-10-21T06:50:01] Info: Synergy Config: C:\Users\yi\AppData\Local\com.rohit\uni_control_hub\synergy.conf
[2024-10-21T06:50:01] Info: Address: 192.168.6.137:25800
[2024-10-21T06:50:01] Info: Server started with pid: 13484
[2024-10-21T06:50:01] NOTE: clipboard sharing is disabled
[2024-10-21T06:50:01] NOTE: started server, waiting for clients
```
 
but then "connect via bluetooth" can not be enabled，nothing added in the log. I clicked it multiple times but nothing happend, and no log output when I click the `connect via bluetooth`. here is my system setup:
```
Edition	Windows 11 Home
Version	22H2
Installed on	‎4/‎18/‎2023
OS build	22621.3593
Experience	Windows Feature Experience Pack 1000.22700.1003.0```

**Comment by rohitsangwan01** [comment]
@Goooyi have you cross checked if Bluetooth radio was on ?

**Comment by Goooyi** [comment]
> @Goooyi have you cross checked if Bluetooth radio was on ?

yes it's on, and I also linked my bluetooth mouse to my pc so the bluetooth of my pc should be working. I also tried disconnected my mouse just in case, but the "connect via bluetooth" feature still not working.

**Comment by rohitsangwan01** [comment]
@Goooyi hard to say without logs, so it would be much helpful if you can run the app from code, and get the logs from console, Btw that `Connect with Bluetooth` feature will work with IOS only for now, because its `Bluetooth low energy` based, and android devices mostly support `Bluetooth Classic` based control, will add that in future
